easybuild/easyconfigs/l/libPSML/libPSML-1.1.12-GCC-11.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'libPSML'
+version = '1.1.12'
+
+homepage = 'https://gitlab.com/siesta-project/libraries/libpsml'
+description = """A library to handle pseudopotentials in PSML format"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://gitlab.com/siesta-project/libraries/libpsml/-/archive/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c61503d5a5de119f970f1bf27aa0ac8059b87e81d1a8013bef1bb7993df44c56']
+
+dependencies = [
+    ('xmlf90', '1.5.6'),
+]
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+]
+
+separate_build_dir = True
```

This can be removed, as it's the default setting.

```suggestion
separate_build_dir = True
```

---

It has been updated